css/main.css

```diff
@@ -689,3 +688,30 @@ a:hover {
   top: 50%;
   margin-top: -10px;
 }
+button .ag-spinner {
+  display: inline-block;
+  font-size: 12pt;
+  padding-right: 10px;
+}
+#results-screen > a > .ag-spinner {
+  z-index: 100;
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  background: rgba(0, 0, 0, 0.3);
+}
+#results-screen > a > .ag-spinner i {
+  left: 50%;
+  margin-left: -25px;
+  margin-top: -25px;
+  font-size: 50px;
+  top: 50%;
+}
+@-moz-keyframes spin { 100% { -moz-transform: rotate(360deg); } }
+@-webkit-keyframes spin { 100% { -webkit-transform: rotate(360deg); } }
+@keyframes spin { 100% { -webkit-transform: rotate(360deg); transform:rotate(360deg); } }
+.spin-icon {
+  -webkit-animation: spin 2s linear infinite;
+  -moz-animation: spin 2s linear infinite;
+  animation: spin 2s linear infinite;
```

Comment:
Not sure if spinners that aren't moving for 3% of the users are really a problem. The fact that the entire application basically isn't working in IE8/9 at all makes an improvement like this pretty much pointless IMO.